Automattic/jetpack
# VideoPress: capital P

`VideoPress` > `Videopress`

Fixes #

#### Changes proposed in this Pull Request:

* VideoPress: capital P

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






Build VideoPress plugin

```cli
jetpack build plugins/videopress
```

* Confirm the Admin page works as expected
* Confirm the `jetpackVideoPressInitialState` global is exposed properly.

<img width="407" alt="image" src="https://user-images.githubusercontent.com/77539/185492694-927c8a16-b975-471b-a61f-1ffb477d3a34.png">
